project/Dependencies.scala

```diff
@@ -22,7 +22,7 @@ object Dependencies {
     val testContainers = "1.15.1"
     val junit = "4.13.1"
     val h2Driver = "1.4.200"
-    val jackson = "2.10.5" // this should match the version of jackson used by akka-serialization-jackson
+    val jackson = "2.10.5.1" // this should match the version of jackson used by akka-serialization-jackson
```

Comment:
Akka 2.6.11 will use this version.

Reply:
We can upgrade akka-projections to akka 2.6.11 and then upgrade jackson, but what if someone is using akka 2.6.10 or earlier? What are the odds that such a patch release will cause trouble? 

Isn't that better for projections to stay on akka 2.6.10 and let it be transitively upgraded by the users?

Reply:
it's used in kafka and examples. Wonder what we use it for in kafka?

Anyway, it should be ok to update to 2.10.5.1 here without updating Akka